### PR TITLE
Add missing cast to cagg union view

### PR DESCRIPTION
**Pull request by svenklemm**
The view definition for the union view was missing a cast from int8
to the actual int type used for hypertables with int partitioning
column this was not a problem on 64 bit systems but on 32 bit systems
int4 and int8 are represented differently.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -90,7 +90,7 @@ jobs:
         - python2 ./scripts/githooks/commit_msg_tests.py
         - python3 ./scripts/githooks/commit_msg_tests.py
 
-    - if: (type = cron) OR (branch = prerelease_test)
+    - if: (type = pull_request) OR (branch = prerelease_test)
```
Comment (gayyappan) [inline .travis.yml:93]: are these changes intended as part of the PR or for testing?

**Comment by svenklemm** [comment]
> code changes look good. Are .yml changes intended for merge?

was just for testing
